spatial/workers/unreal/spatialos.AIWorker.worker.json

```diff
@@ -0,0 +1,104 @@
+{
+  "build": {
+    "tasks": [
+      {
+        "name": "codegen",
+        "description": "required by spatial worker build build-config.",
+        "steps": [{"name": "No-op", "command": "echo", "arguments": ["No-op."]}]
+      },
+      {
+        "name": "build",
+        "description": "required by spatial worker build build-config.",
+        "steps": [{"name": "No-op", "command": "echo", "arguments": ["No-op."]}]
+      },
+      {
+        "name": "clean",
+        "description": "required by spatial worker build build-config.",
+        "steps": [{"name": "No-op", "command": "echo", "arguments": ["No-op."]}]
+      }
+    ]
+  },
+  "bridge": {
+    "worker_attribute_set": {
+      "attributes": [
+        "AIWorker"
+      ]
+    },
+    "entity_interest": {
+      "range_entity_interest": {
+        "radius": 50
+      }
+    },
+    "streaming_query": [
+      {
+        "global_component_streaming_query": {
+          "component_name": "unreal.SingletonManager"
+        }
+      },
+      {
+        "global_component_streaming_query": {
+          "component_name": "unreal.Singleton"
+        }
+      }
+    ],
+    "component_delivery": {
+      "default": "RELIABLE_ORDERED",
+      "checkout_all_initially": true
+    }
+  },
+  "managed": {
+    "windows": {
+      "artifact_name": "AIEditor@Windows.zip",
+      "command": "StartEditor.bat",
+      "arguments": [
+        "ThirdPersonExampleMap",
+        "-server",
+        "-stdout",
+        "-nowrite",
+        "-unattended",
+        "-nologtimes",
+        "-nopause",
+        "-messaging",
+        "-SaveToUserDir",
+        "+workerType",
+        "AIWorker",
+        "+appName",
+        "${IMPROBABLE_PROJECT_NAME}",
+        "+receptionistHost",
+        "${IMPROBABLE_RECEPTIONIST_HOST}",
+        "+receptionistPort",
+        "${IMPROBABLE_RECEPTIONIST_PORT}",
+        "+workerType",
+        "${IMPROBABLE_WORKER_NAME}",
+        "+workerId",
+        "${IMPROBABLE_WORKER_ID}",
+        "+linkProtocol",
+        "RakNet",
```

Comment:
GDK team I believe has switched to TCP by default in 0.3.0, and is planning to move to KCP for clients in 0.4.0. Might be worth syncing with them about this to determine if it might change the profiling results at all.